core/src/test/scala-3/cats/derived/ADTs.scala

```diff
@@ -0,0 +1,378 @@
+/*
+ * Copyright (c) 2015 Miles Sabin
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.derived
+
+import cats.{Eq, Eval}
+import cats.syntax.all.given
+import org.scalacheck.rng.Seed
+import org.scalacheck.{Arbitrary, Cogen, Gen}
+import scala.annotation.tailrec
+
+object ADTs:
```

Comment:
So are we dropping testing for sealed traits in favour of enums? I had assumed that we would test both (although that sucks 😂 )

Reply:
We're not dropping anything 🤔 - we just add more tests

Reply:
Oh so we are! 🤦